src/web/components/Core/Tooltip.tsx

```diff
@@ -13,7 +13,10 @@ export function Tooltip({ children, side, align }: TooltipProps) {
   return (
     <TooltipPrimitive.Provider>
       <TooltipPrimitive.Root>
-        <TooltipPrimitive.Trigger className='tooltip-trigger'>
+        <TooltipPrimitive.Trigger
+          className='tooltip-trigger'
+          onClick={(event) => event.preventDefault()}
```

Comment:
I think instead of leaving this as a 'submit' button (the default button type) and then preventing the submit, better to just use type='button' so it doesn't try to submit at all.